tests/backstop_data/engine_scripts/puppet/on-ready.js

```diff
@@ -8,11 +8,7 @@ module.exports = async (page, scenario, vp) => {
 
   await utility.waitForAngular();
 
-  if (!scenario.captureLoadingScreen) {
-    await utility.waitForLoadingComplete(scenario.waitForLoadingComplete);
-  }
-
-  await mouseEventsHelper(page, scenario);
+  await mouseEventsHelper(page, scenario, null, true);
```

Comment:
`null`, and `true` are not very clear. This is the problem of sending a lot of parameters, specially boolean parameters.